test/arbitrary.test.ts

```diff
@@ -236,16 +236,34 @@ describe('Arbitrary tests', () => {
 
   describe('Filtered Arbitraries', () => {
     it('A filtered mapped filtered arbitrary is able to ', () => {
```

Comment:
This is a poor description for a test. 

`it('should be able to...'`

Reply:
I agree with you but it was already like this in the `master` branch. I already fixed the test description. Can you confirm that it is ok now?